sql/moz-fx-data-shared-prod/search_terms_derived/sanitization_job_metadata/metadata.yaml

```diff
@@ -0,0 +1,18 @@
+---
+friendly_name: Sanitized Firefox Search Terms
+owners:
+  - ctroy@mozilla.com
+labels:
+  incremental: true
+description: |-
+  This table tracks the success of sanitation jobs and some aggregate metrics about what got
+  sanitized for use in model evaluation.
```

Comment:
I think this description should include a note about how it gets populated, including a link to the repo containing the code that will be populating this.

Reply:
Looks like this doesn't have a link to the repo but that's probably fine